### PR TITLE
fix: enforce keepAlive headers to be false

### LINKED ISSUE #79: Keep-Alive with repeated HTTP/S requests
#77 made me think about a possible related issue with resuing connections, which could, at the very least, mess up the timings. I *think* it doesn't happen at this time but it seems that'll change in a future version of node, so we need to keep track of that: https://github.com/nodejs/node/pull/43522

### PR DESCRIPTION
**Pull request by ayuhito**
Resolves #79. Adds one test case. I don't think there is anything to do for http/2 since they [removed the relevant headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Keep-Alive) from the spec, right?

### CODE REVIEW

File: src/command/http-command.ts [hunk 1]
```diff
@@ -106,8 +106,11 @@ export const httpCmd = (options: HttpOptions, resolverFn?: ResolverType): Reques
 			downloadLimit: 10_000,
 		},
 		agent: {
-			http: http.globalAgent,
-			https: new https.Agent({maxCachedSessions: 0}),
+			// Ensure Connection: closed header is used - https://nodejs.org/api/http.html#new-agentoptions
+			/* eslint-disable unicorn/prefer-number-properties */
```
Comment (patrykcieszkowski) [inline src/command/http-command.ts:110]: `eslint-disable` disables the rule until the end of the file, or until it stumbless accross `eslint-enable` comment.

Either use `eslint-disable-next-line` or add `eslint-enable` block below the `agent` object.

File: test/unit/cmd/http.test.ts [hunk 2]
```diff
@@ -150,6 +150,23 @@ describe('http command', () => {
 			expect(mockedSocket.emit.lastCall.args[1]).to.have.nested.property('result.rawBody', expectedResult.result.rawBody);
 			expect(mockedSocket.emit.lastCall.args[1]).to.have.nested.property('result.rawHeaders', expectedResult.result.rawHeaders);
 		});
+
+		it('should ensure keepAlive header is disabled', () => {
```
Comment (patrykcieszkowski) [inline test/unit/cmd/http.test.ts:154]: PR https://github.com/jsdelivr/globalping/pull/170 is changing the schema of this query. We might have to wait for it to merge, and then update the schema before adding this test before merging it.

Comment (ayuhito) [reply test/unit/cmd/http.test.ts:154]: Sure, I'll watch the other PR and then adjust accordingly once that's merged 👍

Comment (ayuhito) [reply test/unit/cmd/http.test.ts:154]: Oh it just got merged 👀

Comment (patrykcieszkowski) [reply test/unit/cmd/http.test.ts:154]: :)

File: src/command/http-command.ts (hunk 1 above)
Comment (patrykcieszkowski) [reply src/command/http-command.ts]: @ayuhito this one was actually correct. I misread the second statement.

Both, your previous and the new solutions are fine by me. If you want to revert the comment - feel free to do so.

My bad.

**Review by patrykcieszkowski — APPROVED** [review]

File: src/command/http-command.ts (hunk 1 above)
Comment (ayuhito) [reply src/command/http-command.ts]: It's all good 👍 We can stick with this version.

**Pull request merged**